### PR TITLE
feat(events): mark as favorite

### PR DESCRIPTION
**Pull request by olros**
## Proposed changes
Brukere kan markere arrangementer som favoritter og filterer på dem, samt motta varsel ved påmeldingsstart

Issue number: closes N/A

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] CHANGELOG.md has been updated. [Guide](https://tihlde.slab.com/posts/changelog-z8hybjom)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] The fixtures have been updated if needed (for migrations)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by CJGutz** [comment]
Skulle man hatt at listen med arrangementer viser favoritter selv for de arrangementene som er ferdig? Eventuelt at favoritter slettes etter at et arrangementer er ferdig?

**Comment by olros** [comment]
> Skulle man hatt at listen med arrangementer viser favoritter selv for de arrangementene som er ferdig? Eventuelt at favoritter slettes etter at et arrangementer er ferdig?

Tenkte det gikk fint at de forsvinner siden du kan filtrere  på `expired` og finne dem allikevel

File: app/content/views/event.py [hunk 1]
```diff
@@ -129,6 +129,30 @@ def destroy(self, request, *args, **kwargs):
...
+    @action(
+        detail=True,
+        methods=["get", "put"],
+        url_path="favorite",
+        permission_classes=(IsMember,),
+    )
+    def user_favorite(self, request, pk, *args, **kwargs):
+        event = get_object_or_404(Event, id=pk)
+
+        if request.method == "PUT":
```
[the remark this answers is not in the document]
Comment (olros) [reply app/content/views/event.py:141]: Jo, for det er get også. Hvis det er get så er ikke request.data en ting

[the remark this answers is not in the document]
Comment (Her0elt) [reply app/content/views/event.py:141]: Ahhh leste litt fort ser det nå

**Review by Her0elt — APPROVED** [review]
:100:

**Pull request merged**